### DISCUSSION TITLE
对web支持的怎么样了？

### DISCUSSION THREAD

**Discussion by wang2650** [dotnetcore/Natasha#46] [Q&A]
1 这个还是只能在控制台或者桌面里用吗？
2 引用第三方的dll，怎么使用？

github上有个nettask的c#项目，一个模块一个文件夹，可以调用其中的模块，模块间互相不影响。
这个项目也能达到类似效果吗？我现在用的是csscript

**Comment by NMSAzulX** [comment]
@wang2650 目前 WEB 没什么问题，我们的项目都在用，Natasha 目前是以域来隔离操作，除了系统域之外，其他域可以卸载。如果您想尝试插件功能也可以。 
您说的模块目前来讲我不是很清楚这个在.NET程序里的区分，是不是运行时中的新程序集？
对于不同新功能之间的调用，我建议采取以下方式以便系统回收：
系统域 委托 = 新域 新功能。
如果您不考虑回收，则需要避免程序集名在主域中重复出现。
